### DISCUSSION TITLE
How to get a MSSQL Server provider to target my database instead of master?

### DISCUSSION THREAD

**Discussion by avenmia** [prisma/prisma#17884] [Q&A]
In my .env I have this for my local database connection string:

DATABASE_URL="sqlserver://localhost:1433;database=MyDatabase;integratedSecurity=true;trustServerCertificate=true;"

When I run the following commands, it creates the migration successfully, but pushes to my master database.

```
npx prisma migrate dev
npx prisma db push
```

I don't get any errors. Here is the output from npx prisma db push

```
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Datasource "db": SQL Server database

The database is already in sync with the Prisma schema.
```

Instead of database, I've also tried initialCatalog in the connection string but no luck.

**Comment by nurul3101** [comment]
Hello @avenmia 👋 

Can you please elaborate on what you mean by pushes to master database? Are the migrations not created on the database that is provided in the env file?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
